### PR TITLE
Information on enabling Speculative Execution

### PR DESCRIPTION
**Pull request by ACStoneCL**
### What does this PR fix/introduce?
This introduces details for enabling speculative execution for node operators.

### Additional context
N/A

### Checklist
(Delete any that aren't relevant)

- [x] Docs are successfully building - `yarn install && yarn run build`.
- [x] My changes follow the [Casper docs style guidelines](https://docs.casper.network/resources/contribute-to-docs/).
- [x] All technical procedures have been tested (if you want help with this, mention it in [Reviewers](#reviewers)).

### Reviewers
@ipopescu

### CODE REVIEW

File: source/docs/casper/operators/setup/basic-node-configuration.md [hunk 1]
```diff
@@ -160,6 +160,14 @@ The following default values are specified in the file if you want to change the
 - The port used for networking
 - Known_addresses - these are the bootstrap nodes (there is no need to change these)
 
+### Enabling Speculative Execution
+
+The `speculative_exec` endpoint provides a method to execute a Deploy without committing its execution effects to global state. This can be used by developers to roughly estimate the gas costs of sending the Deploy in question. By default, speculative_exec is disabled on a node.
+
+`speculative_exec` can be enabled within *config.toml* by changing `enable_server` to `true` under the configuration options for the speculative execution JSON-RPC HTTP server.
+
+Node operators may also change the incoming request port for speculative execution, which defaults to `7778`. Further, you can choose to alter the `qps_limit` and `max_body_bytes`, which limit the amount and size of requests to the speculative execution server.
```
[the remark this answers is not in the document]
Comment (ACStoneCL) [reply source/docs/casper/operators/setup/basic-node-configuration.md:169]: This is an excellent point, I've added an enabled configuration example.

[the remark this answers is not in the document]
Comment (ipopescu) [reply source/docs/casper/operators/setup/basic-node-configuration.md:169]: Looks great, @ACStoneCL!

File: source/docs/casper/operators/setup/basic-node-configuration.md [hunk 2]
```diff
@@ -160,6 +160,53 @@ The following default values are specified in the file if you want to change the
...
+### Enabling Speculative Execution
+
+The `speculative_exec` endpoint provides a method to execute a Deploy without committing its execution effects to global state. This can be used by developers to roughly estimate the gas costs of sending the Deploy in question. By default, `speculative_exec` is disabled on a node.
+
+`speculative_exec` can be enabled within *config.toml* by changing `enable_server` to `true` under the configuration options for the speculative execution JSON-RPC HTTP server.
+
+Node operators may also change the incoming request port for speculative execution, which defaults to `7778`. Further, you can choose to alter the `qps_limit` and `max_body_bytes`, which limit the amount and size of requests to the speculative execution server.
+
+<details>
+<summary><b>Example Config.toml configuration with speculative execution enabled</b>b></summary>
```
Comment (andrzej-casper) [inline source/docs/casper/operators/setup/basic-node-configuration.md:172]: This is the reason why build is failing:

```suggestion
<summary><b>Example Config.toml configuration with speculative execution enabled</b></summary>
```

Comment (ACStoneCL) [reply source/docs/casper/operators/setup/basic-node-configuration.md:172]: Thanks andrzej! I'm facepalming pretty hard right now.

**Pull request merged**